### ISSUE TITLE
Installing latest version on magneto 2.3.3 fails.

### ISSUE THREAD

**Issue by mohdali27** [tig-nl/postnl-magento2#143]
composer require tig/postnl-magento2
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - magento/framework 100.1.15 requires php ~5.6.5|7.0.2|7.0.4|~7.0.6 -> your PHP version (7.1.33) does not satisfy that requirement.
    - magento/framework 100.1.14 requires php ~5.6.5|7.0.2|7.0.4|~7.0.6 -> your PHP version (7.1.33) does not satisfy that requirement.
    - magento/framework 100.1.13 requires php ~5.6.5|7.0.2|7.0.4|~7.0.6 -> your PHP version (7.1.33) does not satisfy that requirement.
    - magento/framework 100.1.12 requires php ~5.6.5|7.0.2|7.0.4|~7.0.6 -> your PHP version (7.1.33) does not satisfy that requirement.
    - magento/framework 100.1.11 requires php ~5.6.5|7.0.2|7.0.4|~7.0.6 -> your PHP version (7.1.33) does not satisfy that requirement.
    - magento/framework 100.1.10 requires php ~5.6.5|7.0.2|7.0.4|~7.0.6 -> your PHP version (7.1.33) does not satisfy that requirement.
    - magento/framework 100.1.9 requires php ~5.6.5|7.0.2|7.0.4|~7.0.6 -> your PHP version (7.1.33) does not satisfy that requirement.
    - magento/framework 100.1.8 requires php ~5.6.5|7.0.2|7.0.4|~7.0.6 -> your PHP version (7.1.33) does not satisfy that requirement.
    - magento/framework 100.1.7 requires php ~5.6.5|7.0.2|7.0.4|~7.0.6 -> your PHP version (7.1.33) does not satisfy that requirement.
    - magento/framework 100.1.6 requires php ~5.6.5|7.0.2|7.0.4|~7.0.6 -> your PHP version (7.1.33) does not satisfy that requirement.
    - magento/framework 100.1.5 requires php ~5.6.5|7.0.2|7.0.4|~7.0.6 -> your PHP version (7.1.33) does not satisfy that requirement.
    - magento/framework 100.1.4 requires php ~5.6.5|7.0.2|7.0.4|~7.0.6 -> your PHP version (7.1.33) does not satisfy that requirement.
    - magento/framework 100.1.3 requires php ~5.6.5|7.0.2|7.0.4|~7.0.6 -> your PHP version (7.1.33) does not satisfy that requirement.
    - magento/framework 100.1.2 requires php ~5.6.5|7.0.2|7.0.4|~7.0.6 -> your PHP version (7.1.33) does not satisfy that requirement.
    - magento/framework 100.1.1 requires php ~5.6.0|7.0.2|~7.0.6 -> your PHP version (7.1.33) does not satisfy that requirement.
    - magento/framework 100.1.0 requires php ~5.6.0|7.0.2|~7.0.6 -> your PHP version (7.1.33) does not satisfy that requirement.
    - Conclusion: don't install tig/postnl-magento2 1.8.1
    - Installation request for zendframework/zend-barcode (locked at 2.8.0) -> satisfiable by zendframework/zend-barcode[2.8.0].
    - Installation request for magento/product-community-edition 2.3.3 -> satisfiable by magento/product-community-edition[2.3.3].
    - magento/framework 101.0.1 requires colinmollenhour/php-redis-session-abstract 1.3.4 -> satisfiable by colinmollenhour/php-redis-session-abstract[v1.3.4].
    - magento/framework 101.0.2 requires colinmollenhour/php-redis-session-abstract 1.3.4 -> satisfiable by colinmollenhour/php-redis-session-abstract[v1.3.4].
    - magento/framework 101.0.3 requires colinmollenhour/php-redis-session-abstract 1.3.4 -> satisfiable by colinmollenhour/php-redis-session-abstract[v1.3.4].
    - magento/framework 101.0.4 requires colinmollenhour/php-redis-session-abstract 1.3.4 -> satisfiable by colinmollenhour/php-redis-session-abstract[v1.3.4].
    - magento/framework 101.0.5 requires colinmollenhour/php-redis-session-abstract 1.3.4 -> satisfiable by colinmollenhour/php-redis-session-abstract[v1.3.4].
    - magento/framework 101.0.6 requires colinmollenhour/php-redis-session-abstract 1.3.4 -> satisfiable by colinmollenhour/php-redis-session-abstract[v1.3.4].
    - magento/framework 101.0.7 requires colinmollenhour/php-redis-session-abstract 1.3.4 -> satisfiable by colinmollenhour/php-redis-session-abstract[v1.3.4].
    - magento/framework 101.0.8 requires colinmollenhour/php-redis-session-abstract 1.3.4 -> satisfiable by colinmollenhour/php-redis-session-abstract[v1.3.4].
    - magento/framework 101.0.9 requires colinmollenhour/php-redis-session-abstract 1.3.4 -> satisfiable by colinmollenhour/php-redis-session-abstract[v1.3.4].
    - Conclusion: don't install colinmollenhour/php-redis-session-abstract v1.3.4
    - Conclusion: don't install magento/framework 101.0.0
    - Conclusion: don't install tig/postnl-magento2 1.8.0.1|install magento/framework 101.0.0|install colinmollenhour/php-redis-session-abstract v1.3.4
    - Conclusion: don't install tig/postnl-magento2 1.8.0|remove magento/framework 102.0.3|install magento/framework 101.0.0|install magento/framework 101.0.1|install magento/framework 101.0.2|install magento/framework 101.0.3|install magento/framework 101.0.4|install magento/framework 101.0.5|install magento/framework 101.0.6|install magento/framework 101.0.7|install magento/framework 101.0.8|install magento/framework 101.0.9
    - Conclusion: don't install tig/postnl-magento2 1.8.0|don't install magento/framework 102.0.3|install magento/framework 101.0.0|install magento/framework 101.0.1|install magento/framework 101.0.2|install magento/framework 101.0.3|install magento/framework 101.0.4|install magento/framework 101.0.5|install magento/framework 101.0.6|install magento/framework 101.0.7|install magento/framework 101.0.8|install magento/framework 101.0.9
    - Installation request for tig/postnl-magento2 ^1.8 -> satisfiable by tig/postnl-magento2[1.8.0, 1.8.0.1, 1.8.1].
    - tig/postnl-magento2 1.8.0 requires magento/framework >=100.1.0,<=100.1.18|>=101.0.0,<=101.0.9|>=102.0.0,<=102.0.2 -> satisfiable by magento/framework[100.1.0, 100.1.1, 100.1.2, 100.1.3, 100.1.4, 100.1.5, 100.1.6, 100.1.7, 100.1.8, 100.1.9, 101.0.0, 101.0.1, 100.1.10, 100.1.11, 101.0.2, 100.1.12, 101.0.3, 100.1.13, 101.0.4, 101.0.5, 100.1.14, 100.1.15, 101.0.6, 100.1.16, 101.0.7, 102.0.0, 100.1.17, 102.0.1, 101.0.8, 102.0.2, 100.1.18, 101.0.9].
    - Can only install one of: magento/framework[102.0.3, 100.1.16].
    - Can only install one of: magento/framework[102.0.3, 102.0.0].
    - Can only install one of: magento/framework[102.0.3, 100.1.17].
    - Can only install one of: magento/framework[102.0.3, 102.0.1].
    - Can only install one of: magento/framework[102.0.3, 102.0.2].
    - Can only install one of: magento/framework[102.0.3, 100.1.18].
    - Installation request for magento/framework (locked at 102.0.3) -> satisfiable by magento/framework[102.0.3].

I'm trying to install the latest version using composer with magneot community version 2.3.3, and I'm receiving the error above.

**Comment by tig-dennisvanderhammen** [comment]
Hi @mohdali27,

PostNL version 1.8.1 is compatible with Magento 2.3.3. This might be blocked in your environment because of a set version of a specific module. Could you please show us your composer.json?

**Comment by GuiltyNL** [comment]
I have had the same issue on a new clean composer installation of Magento 2.3.3. My solution was to run it like this:
```

composer require tig/postnl-magento2 --no-update
composer update
```

**Comment by tig-dennisvanderhammen** [comment]
Hi @GuiltyNL,

Thank you for providing a solution as well. 

The issue was most likely the zendframework/zend-barcode which was locked at version 2.8.0 and we required either 2.5.2 or 2.7.0.

We have fixed this issue in version v1.9.0 which was released on 28th of januari. From this version and on we will accept more zend-barcode versions.
